packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -449,6 +454,20 @@ class UserSessionManager implements IUserSessionManager {
       );
     });
 
+    const clearInProgressFlags = () => {
+      sessionKeys.forEach((sessionKey: UserSessionKey) => {
```

Comment:
Is it possible that sessionKeys got updated as well and UserSessionKey is not present. so serverSideCookiesRequestInProgress will still contain stale values?